File: pset1/Modulo01 - C 🔋/mario/Mario_V2.c
#include <stdio.h>

int main() {
    int altura, linha, espacos, hashtags;

    printf("Digite um número entre 1 e 8: ");
    scanf("%d", &altura);

    // Verifica se a altura está dentro do intervalo válido
    if (altura < 1 || altura > 8) {
        printf("Altura inválida. O número deve estar entre 1 e 8.\n");
        return 1;
    }

    for (linha = 1; linha <= altura; linha++) {
        // Imprime os espaços em branco antes dos hashtags da primeira pirâmide
        for (espacos = altura - linha; espacos > 0; espacos--) {
            printf(" ");
        }

        // Imprime os hashtags da primeira pirâmide
        for (hashtags = 1; hashtags <= linha; hashtags++) {
            printf("#");
        }

        // Adiciona os espaços entre as pirâmides
        printf("  ");

        // Imprime os hashtags da segunda pirâmide
        for (hashtags = 1; hashtags <= linha; hashtags++) {
            printf("#");
        }

        printf("\n");
    }

    return 0;
}

File: pset1/Modulo06 - Python 🐍/mario/Mario_v2.py
altura = int(input("Digite um número entre 1 e 8: "))

# Verifica se a altura está dentro do intervalo válido
if altura < 1 or altura > 8:
    print("Numero inválida. O número deve estar entre 1 e 8.")
    exit(1)

for linha in range(1, altura + 1):
    # Imprime os espaços em branco antes dos hashtags da primeira pirâmide
    for espacos in range(altura - linha, 0, -1):
        print(" ", end="")

    # Imprime os hashtags da primeira pirâmide
    for hashtags in range(1, linha + 1):
        print("#", end="")

    # Adiciona os espaços entre as pirâmides
    print("  ", end="")

    # Imprime os hashtags da segunda pirâmide
    for hashtags in range(1, linha + 1):
        print("#", end="")

    print()
